src/transformers/modeling_xlnet.py

```diff
@@ -1040,10 +1296,11 @@ def forward(
         input_mask=None,
         head_mask=None,
         inputs_embeds=None,
+        labels=None,
```

Comment:
Reorder for consistency

Reply:
One note here is that torchscript use to convert inputs in tuples of arguments and couldn't split these tuples very easily so the inputs of the inner model used to have to be a compact subset of the inputs of the outer model.

Cf this issue for details: https://github.com/huggingface/transformers/issues/1010

I'm not sure it's still the case, but it's probably good to check this.

Reply:
The torchscript tests still all pass. Not sure if there is another way to see if this breaks something.

Reply:
Double-checked and the arguments passed to the inner model are before the reordering, so this should not have any impact.